rest-server/src/config/job.js

```diff
@@ -74,15 +74,22 @@ const jobConfigSchema = Joi.object().keys({
           .default([]),
         command: Joi.string()
           .required(),
+        minFailedTaskCount: Joi.number()
+          .integer()
+          .min(1)
+          .allow(null)
+          .default(1),
```

Comment:
Can you test below things:

If user set the minFailedTaskCount = null, from below line:

'minFailedTaskCount': data.taskRoles[i].minFailedTaskCount,

What will you send to launcher? 
I expect you also send null to launcher.